ls1intum/Artemis
# `Programming exercises`: Do not display submission policy to student if not active

### Checklist
#### General


#### Client

### Motivation and Context


Fix #6719

### Description

Only show the submission policy to the student in the exercise header and online editor when the policy is activated.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise with a submission policy that is not activated

1. Log in to Artemis
2. Create a programming exercise with no submission policy.
3. Add a submission policy to the programming exercise in the Grading configuration but do not activate the policy.
4. Participate in the exercise using the online editor.
5. Notice the submission count and that you can submit after reaching the limit.

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


See issue
